app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    our_list = []
+    for human in people:
+        people = Person(human["name"], human["age"])
+        if "husband" in human:
+            if human["husband"] is not None:
+                people.husband = human["husband"]
+        if "wife" in human:
+            if human["wife"] is not None:
```

Comment:
Similar to the previous comment, you can combine these two conditions using `and` to avoid nested `if` statements.